src/helm_values.tf

```diff
@@ -0,0 +1,18 @@
+locals {
+  helm_values = {
+    nameOverride = var.md_metadata.name_prefix
+    podLabels    = var.md_metadata.default_tags
+    replicaCount = var.replica_configuration.replicas
+    resources = {
+      requests = {
```

Comment:
Usually requests is less than limits - like 1/2 or 2/3 of the limit. You can make them the same, but it effectively guarantees that you'll either be CPU throttled, have OOMKilled pods, or be over-provisioned.